### PR TITLE
Fix/filter reload

### PR DESCRIPTION
**Pull request by mingard**

### CODE REVIEW

**Comment by mingard** [comment]
@eduardoboucas this is not ready to review

File: frontend/components/DocumentFilters/DocumentFilters.jsx [hunk 1]
```diff
@@ -24,36 +24,61 @@ export default class DocumentFilters extends Component {
...
   constructor(props) {
     super(props)
 
-    const {collection, filter} = props
+    const {collection} = this.props
+    const paramFilters = this.setFiltersFromParams()
+    this.state.filters = paramFilters.length ? paramFilters : this.createDefaultFilters(collection)
+  }
+
+  setFiltersFromParams () {
```
Comment (eduardoboucas) [inline frontend/components/DocumentFilters/DocumentFilters.jsx]: Shouldn't this be called `getFiltersFromParams()`?

File: frontend/components/DocumentFilters/DocumentFilters.jsx [hunk 2]
```diff
@@ -24,36 +24,61 @@ export default class DocumentFilters extends Component {
...
+    const {collection, visible} = this.props
+
+    if (!visible) return null
 
     return (
       <form class={styles.filters} onSubmit={e => e.preventDefault()}>
-        {filters && collection && filters.map((filter, index) => (
+        {filters && collection && filters.map((filter, index) => {
+          return (
           <DocumentFilter
```
Comment (eduardoboucas) [inline frontend/components/DocumentFilters/DocumentFilters.jsx]: I'd indent `DocumentFilter` inside the `return`.

**Comment by eduardoboucas** [comment]
Just added two tiny comments. Looks good to me.

File: frontend/components/DocumentFilters/DocumentFilters.jsx (hunk 1 above)
Comment (mingard) [inline frontend/components/DocumentFilters/DocumentFilters.jsx]: @eduardoboucas indeed it should

File: frontend/components/DocumentFilters/DocumentFilters.jsx (hunk 2 above)
Comment (mingard) [inline frontend/components/DocumentFilters/DocumentFilters.jsx]: I meant to remove the return, it was there for testing so I could output filter to the console. Sorted now

**Pull request merged**